modules/jaeger-rn-new-features.adoc

```diff
@@ -3,12 +3,26 @@ Module included in the following assemblies:
 - rhbjaeger-release-notes.adoc
 ////
 ////
-Feature – Describe the new functionality available to the customer.  For enhancements, try to describe as specifically as possible where the customer will see changes.
-Reason – If known, include why has the enhancement been implemented (use case, performance, technology, etc.).   For example, showcases integration of X with Y, demonstrates Z API feature, includes latest framework bug fixes.
+Feature – Describe the new functionality available to the customer. For enhancements, try to describe as specifically as possible where the customer will see changes.
+Reason – If known, include why has the enhancement been implemented (use case, performance, technology, etc.). For example, showcases integration of X with Y, demonstrates Z API feature, includes latest framework bug fixes.
 Result – If changed, describe the current user experience.
 ////
 
 [id="jaeger-rn-new-features_{context}"]
+== New features {ProductName} 1.24.0
+
+This release of {ProductName} includes the following new features:
+
+* Adds support for the Red Hat Elasticsearch Operator 5.0 and 5.1.
+
+* Adds Elasticsearch rollover support.
+
+* Adds Elasticsearch tolerations support.
+
+* Updates the Jaeger Operator to include support for Strimzi Kafka Operator 0.23.0. A known issue and workaround associated with this update is documented in the "Known Issues" section.
```

Comment:
Is there a reason that mostly the pattern "the [whatever] Operator" is used, but for Strimzi Kafka Operator the article "the" is omitted? (Not just here, in all instances)

Reply:
Fixed.